src/Input/Input.jsx

```diff
@@ -25,13 +38,15 @@ Input.propTypes = {
   type: PropTypes.string,
   placeholder: PropTypes.string,
   label: PropTypes.string,
-  optional: PropTypes.bool
+  optional: PropTypes.bool,
+  iconName: PropTypes.string
 };
 
 Input.defaultProps = {
   type: "",
   name: "default",
   placeholder: "",
   label: "",
-  optional: false
+  optional: false,
+  iconName: undefined
```

Comment:
Make this “” instead of undefined. Empty strings are falsy. It’s just a bit more understandable.